KEYCLOAK-12125 Update docs regarding SameSite changes (#824)
keycloak/keycloak-documentation: release_notes/topics/9_0_0.adoc

extension. The creation and management of the credential store is handled by Elytron using either the `elytron` subsystem or the
`elytron-tool.sh` script.

> == SameSite Cookie changes with upcoming Google Chrome update

Reviewer: ```suggestion
== SameSite cookie changes with upcoming Google Chrome update
```

> Google chrome starting version 80, will change its default value for SameSite cookie argument to `Lax`. Therefore we

Reviewer: ```suggestion
Google Chrome starting with version 80 will change the default value for `SameSite` cookie parameter to `Lax`. Therefore
```

> needed to change some of our cookies (especially those which are used within our Javascript adapter for checking session

Reviewer: ```suggestion
changes were required to several {project_name} cookies (especially those which are used within our Javascript adapter for checking session
```

> status using the iframe) to have SameSite argument set to None which requires also setting the argument Secure. This

Reviewer: ```suggestion
status using the iframe) to set SameSite parameter to `None`. Please note that this settings also requires setting the Secure parameter, hence
```

> means starting with version 9.0.0 our Javascript adapter will be fully functional only using TLS on {project_name} side.

Reviewer: ```suggestion
starting with this version, our Javascript adapter will only be fully functional when using SSL / TLS connection on {project_name} side.
```

extension. The creation and management of the credential store is handled by Elytron using either the `elytron` subsystem or the
`elytron-tool.sh` script.

== SameSite cookie changes with upcoming Google Chrome update

> Google Chrome starting with version 80, will change the default value for `SameSite` cookie parameter to `Lax`.

Reviewer: ```suggestion
Starting with version 80, Google Chrome will change the default value for `SameSite` cookie parameter to `Lax`.
```
